Maths/CountDigit.java

```diff
@@ -0,0 +1,13 @@
+import java.util.*;
+import java.lang.*;
+// count the number of digits in a number
+class CountDigit{
+	public static void main(String args[]){
+		Scanner sc = new Scanner(System.in);
+		System.out.print("Enter the number: ");
+		int number = sc.nextInt();
+		int digits = 0;
+		digits = (int)Math.floor(Math.log10(number) + 1);
```

Comment:
@Utsav1999  Its a good thinking but it has some limitations as it does not work for negetive numbers and zero paticularly because the log function is defined in (0,inf)

Reply:
Then I will add two if conditions, one for checking whether it is 0 or not and the other one for checking the negative number.

Reply:
You can try absolute fn and an if else in case of 0

Reply:
@deadshotsb I updated my code.